mindjunkies/forums/tests.py

```
@@ -0,0 +1,88 @@
+    def test_forum_home_view_requires_login(self):
+        response = self.client.get(reverse("forum_home", kwargs={"course_slug": self.course.slug}))
+        self.assertEqual(response.status_code, 302)  # Should redirect to login
+
+    def test_forum_home_view_authenticated(self):
+        self.client.login(username="testuser", password="testpass")
+        response = self.client.get(reverse("forum_home", kwargs={"course_slug": self.course.slug}))
+        self.assertEqual(response.status_code, 200)  # Should render the forum home
+
+    def test_forum_thread_view(self):
+        self.client.login(username="testuser", password="testpass")
+        response = self.client.get(reverse("forum_thread", kwargs={"course_slug": self.course.slug, "module_id": self.module.id}))
```

Ruff (0.8.2)

Line too long (130 > 119)

(E501)